src/components/table/table.tsx

```
@@ -198,6 +218,7 @@ export class Table {
     private pool: ElementPool;
     private columnFactory: ColumnDefinitionFactory;
     private firstRequest: boolean;
+    private initialized: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)